dataprofiler/tests/profilers/test_datalabeler_options.py

```diff
@@ -0,0 +1,126 @@
+from dataprofiler.profilers.profiler_options import DataLabelerOptions
+from dataprofiler.tests.profilers.test_base_column_options import TestBaseColumnOptions
+from dataprofiler.labelers.base_data_labeler import BaseDataLabeler
+
+
+class TestDataLabelerOptions(TestBaseColumnOptions):
+    
+    option_class = DataLabelerOptions
+
+    def test_init(self):
+        options = self.get_options()
+        expected_val = {'data_labeler_dirpath': None, 
+            'max_sample_size': None, 
+            'is_enabled': True,
+            'data_labeler_object': None}
+
+        self.assertDictEqual(expected_val, options.properties)    
+    
+    def test_set_helper(self):
+        super().test_set_helper()
+
+    def test_set(self):
+        super().test_set()
+    
+    def test_validate_helper(self):
+        super().test_validate_helper()
+        optpth = self.get_options_path() 
+
+        # Test valid dirpath
+        options = self.get_options()
+        options.set({'data_labeler_dirpath': ''})
+        self.assertEqual([], options._validate_helper())
```

Comment:
so, _validate_helper returns empty list if there's no error? If so, can you add a comment here for the clarity

Reply:
Added a comment to both test_validate and test_validate_helper that should help clarify this.